packages/election/graphql/resolvers/_mutations/submitCandidacy.js

```diff
@@ -0,0 +1,42 @@
+const getElections = require('../../../lib/getElections')
+const resolveCandidate = require('../../../lib/resolveCandidate')
+const { upsert } = require('../../../lib/db')
+
+const {
+  Roles,
+  ensureSignedIn
+} = require('@orbiting/backend-modules-auth')
+
+module.exports = async (_, { slug }, { pgdb, user: me, req }) => {
+  ensureSignedIn(req)
+  Roles.ensureUserIsInRoles(me, ['admin', 'associate'])
+
+  const election = (await getElections(pgdb, me, {slug}))[0]
+
+  if (!election) {
+    throw new Error(`No election for slug ${slug}`)
+  }
+
+  const comment = await upsert(
```

Comment:
Here is a third version of the `upsert` formatting version and I feel like a jerk. (see https://github.com/orbiting/backends/pull/83/files#r220564191)

In fact, I'd go for either trailing whitespaces (or not):

```
{ userId: me.id, discussionId: election.discussion.id }
```